### DISCUSSION TITLE
How to import flowchart from diagrams in Python

### DISCUSSION THREAD

**Discussion by animesh-11** [mingrammer/diagrams#829] [Q&A] [answered]
I am trying to import flowchart but its giving error in Jupyter Notebook. Can anyone please help if I am doing in wrong way.

from diagrams import Diagram
from diagrams.programming.flowchart import Inspection
from diagrams.programming.flowchart import Action
from diagrams.programming.flowchart import Decision

with Diagram("First BPMN Diag") as diag:
    start = Inspection("Start")
    task1 = Action("Hello World")
    task2 = Action("Good Bye")
    end = Inspection("End")
    start >> task1 >> task2 >> end
diag

**Comment by animesh-11** [comment]
This is the error coming :
ModuleNotFoundError: No module named 'diagrams.programming.flowchart'

For other modules , its working fine. Pls help.

**Comment by clayms** [comment, marked as answer]
Your code works as-is for me when run from a Jupyter notebook.
![image](https://user-images.githubusercontent.com/12793783/211338742-516d7284-2604-4982-bbaa-f2601df85612.png)

It looks like you might have a version issue.  Try updating diagrams.


You could add `show=False` to the `Diagram` parameters.  This way it will not try to open an external viewer. 

e.g.:
```python
#...
with Diagram("First BPMN Diag", show=False) as diag:
#...
```

**Reply by animesh-11** [reply]
Thanks for the help @clayms :)
I went through the folder env\Lib\site-packages\diagrams\programming and there was no "flowchart.py" source file as well as there was no folder with images  with flowchart folder in the path env\Lib\site-packages\resources\programming and hence it was throwing error. I tried to pip install two times but it didn't help. So finally I copied the source files and image folder directly from this Github repository and the code ran :)
